src/org/zaproxy/zap/extension/websocket/ExtensionWebSocket.java

```diff
@@ -191,6 +197,7 @@ public void init() {
 		super.init();
 		
 		allChannelObservers = new Vector<>();
+		allChannelSenderListeners = new Vector<>();
```

Comment:
Could use `ArrayList` (if we really want to sync we can use `Collections.synchronizedList` and make sure we sync when iterating).